wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -22,39 +22,44 @@
 
 #define __ESP32_CRYPT_H__
 
+#ifdef WOLFSSL_USER_SETTINGS
```

Comment:
Please include wolfssl/wolfcrypt/settings.h always instead. It does this logic at top and ensures build options checks are done.

Reply:
that's good to know, thanks. fixed.